src/components/Sing/ScoreSequencer.vue

```diff
@@ -614,17 +682,26 @@ export default defineComponent({
         const position = note.position - snapTicks.value;
         editedNotes.push({ ...note, position });
       }
-      if (editedNotes.some((note) => note.position < 0)) {
+      if (
+        editedNotes.length === 0 ||
+        editedNotes.some((note) => note.position < 0)
+      ) {
         return;
       }
       store.dispatch("UPDATE_NOTES", { notes: editedNotes });
     };
 
     const handleNotesBackspaceOrDelete = () => {
+      if (state.selectedNoteIds.size === 0) {
+        return;
+      }
       store.dispatch("REMOVE_SELECTED_NOTES");
     };
 
-    const handleNotesKeydown = (event: KeyboardEvent) => {
+    const handleKeydown = (event: KeyboardEvent) => {
+      if (!nowPreviewing.value) {
+        return;
+      }
```

Comment:
逆かも？　意図も書いておいてあげると親切かもです。
```suggestion
      // プレビュー中の操作は想定外の挙動をしそうなので防止
      if (nowPreviewing.value) {
        return;
      }
```

Reply:
これはたぶん逆だと思うので適用させていただきます！　間違ってたらすみません！！